test-app/src/TsApp.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
+const styles = require('./App.css')
+
+const TsApp : (() => JSX.Element) = () => 
```

Comment:
I suggest we go all the way here. If we have a TypeScript-based example then let's drop the JS one.
And we can rename `TsApp` => `App` then 😉